### PR TITLE
fallout with PH

### PR DESCRIPTION
**Pull request by Rayfod**
## About The Pull Request
Fermichems were removed from fallout chems. There is not really anything else to say about it.
Hydrogen was balanced with its PH, so it does not ruin all recipes that use it.

## Why It's Good For The Game

Previously, you had to sit down and code dive even to know the basics of how to make fallout chems. That is, of course, not an ideal situation for the players and causes a lot of frustration. Even more, when fallout chems were the only ones that had to suffer with this.

## Pre-Merge Checklist

- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog

:cl:
del: Removed fermichems on fallout chems + PH on hydrogen
/:cl:

### CODE REVIEW

**Comment by ProbablyCarl** [comment]
Utilize comment blocks rather than single line comments.
To do so, you do /* and */ to open and close them, respectively.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1079,7 +1079,7 @@
 	boiling_point = 20.271
 	color = "#808080" // rgb: 128, 128, 128
 	taste_mult = 0
-	pH = 0.1//Now I'm stuck in a trap of my own design. Maybe I should make -ve pHes? (not 0 so I don't get div/0 errors)
+	pH = 7//Now I'm stuck in a trap of my own design. Maybe I should make -ve pHes? (not 0 so I don't get div/0 errors)
```
Comment (ProbablyCarl) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:1082]: This is the only thing I'm hesitant on. Why are we changing this?
Isn't this correct?

Comment (IrkallaEpsilon) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:1082]: Ah there the github is, carl check dms I gave a lengthy explanation. Tiny nitpick. pH for hydrogen should be removed completely tbf as well as for several other substances that have pH values assigned to them. pH isnt really fully applicable. The chemistry system from Fermi means well. It just has some weird fucky things in it if you worked a bunch in that field.

**Review by ProbablyCarl — APPROVED** [review]

**Comment by Pokebutt** [comment]
super stims no longer work at all

**Comment by Rayfod** [comment]
#420 Fixes the stimpack recipe

**Pull request merged**